distributions/Jamulus.1

```diff
@@ -227,48 +227,47 @@ left to right.
 does not provide feedback as to the current state of the Solo and Mute
 buttons so the controller must track and signal their state locally.
 .It Fl \-directoryfile Ar file
-.Pq directory server mode only
-enable server list persistence, storing to and loading from
-.Ar file
+.Pq Directory Server mode only
+remember registered Servers even if the Directory is restarted
 .It Fl \-mutemyown
-.Pq headless client only
+.Pq headless Client only
 mute my channel in my personal mix
 .It Fl \-norecord
-.Pq server mode only
+.Pq Server mode only
 do not automatically start recording even if configured with
 .Fl R
 .It Fl \-serverbindip Ar ip
-.Pq server mode only
+.Pq Server mode only
 configure Legacy IP address to bind to
 .It Fl \-serverpublicip Ar ip
-.Pq server mode only
-configure public Legacy IP address when both the directory server
-and the actual server are situated behind the same NAT, so that
-clients can connect
+.Pq Server mode only
+configure public legacy IP address when both the Directory Server
+and the actual Server are situated behind the same NAT, so that
+Clients can connect
 .It Fl \-showallservers
-.Pq client mode only
-show all registered servers in the serverlist regardless whether a ping
-to the server is possible or not
+.Pq Client mode only
+show all registered Servers in the serverlist regardless whether a ping
+to the Server is possible or not
 .Pq debugging command
 .It Fl \-showanalyzerconsole
-.Pq client mode only
+.Pq Client mode only
 show analyser console to debug network buffer properties
 .Pq debugging command
 .El
 .Pp
 Note that the debugging commands are not intended for general use.
 .Pp
 .Nm Jamulus
-knows four modes of operation: client mode and three kinds of server
-.Pq private , public , directory .
-A private server is unlisted, clients can only connect if given
+knows four modes of operation: Client mode and three kinds of Server
+.Pq Unregistered, Registered, Directory.
+A Unregistered Server is unlisted, Clients can only connect if given
```

Comment:
> the address (IP address and port).

Not necessarily "IP address" - the host name is perfectly acceptable, if available.  Port number is only necessary when it's not the default value.  This can be covered here.  Or link to the website for further discussion of "Server Address".